src/Components/CommentCard.jsx

```diff
@@ -1,16 +1,66 @@
-import axios from "axios"
-import { useState } from "react"
+import axios from "axios";
+import { useState } from "react";
 
-function CommentCard({comment}){
-    const [vote, setVotes] = useState(comment.votes)
-    return(
-         <>
-             <h3>{comment.author}</h3>
-             <p>{comment.created_at}</p>
-             <p className="comment-text">{comment.body}</p>
-             <p className="votes">Votes: {vote}</p>
-         </>
-    )
+function CommentCard({ comment }) {
+  const [vote, setVotes] = useState(comment.votes);
+  const voteTracker = {
+    inc_votes: 1,
+  };
+
+  function addVote() {
+    voteTracker.inc_votes = 1
+  }
+  function minusVote() {
+    voteTracker.inc_votes = -1
+  }
+
+  function modifyVotes() {
```

Comment:
could you call this function with a number - either `1` or `-1`, instead of having a vote tracker?